### PR TITLE
ISPN-12665 Add cache name to peristence migration logs

### PR DESCRIPTION
**Pull request by ryanemerson**
https://issues.redhat.com/browse/ISPN-12665
https://issues.redhat.com/browse/ISPN-12671

### CODE REVIEW

File: persistence/jdbc/src/main/java/org/infinispan/persistence/jdbc/stringbased/JdbcStringBasedStore.java [hunk 1]
```diff
@@ -148,14 +148,14 @@ public void start() {
                tableManager.updateMetaTable(connection);
             } else {
                // The meta table does not exist, therefore we must be reading from a 11.x store. Migrate the old data
-               org.infinispan.util.logging.Log.PERSISTENCE.startMigratingPersistenceData();
+               org.infinispan.util.logging.Log.PERSISTENCE.startMigratingPersistenceData(cacheName);
```
Comment (pruivo) [inline persistence/jdbc/src/main/java/org/infinispan/persistence/jdbc/stringbased/JdbcStringBasedStore.java:151]: offtopic:
can you remove the following fields? they are only used in the `start()` method and can be obtained from `ctx` and it saves some memory:
`globalConfiguration`
`cacheName`
`isDistributedCache`

Saving (computed by JOL plugin):
compressed oops: 64 bytes => 56 bytes
8bytes align: 120 bytes => 96  bytes

**Review by pruivo — COMMENTED** [review]
It looks good to me. I have just an optional comment. 
I can merge it right away if you want.

File: persistence/jdbc/src/main/java/org/infinispan/persistence/jdbc/stringbased/JdbcStringBasedStore.java (hunk 1 above)
Comment (ryanemerson) [reply persistence/jdbc/src/main/java/org/infinispan/persistence/jdbc/stringbased/JdbcStringBasedStore.java:145]: Done

**Review by pruivo — APPROVED** [review]

**Comment by pruivo** [comment]
integrated! thanks @ryanemerson !
are you going to open a PR for 12.0.x?

**Comment by ryanemerson** [comment]
> are you going to open a PR for 12.0.x?

I didn't think it was worth it with it only being a log change and 12.1 around the corner.

**Comment by pruivo** [comment]
Just wanted to make sure I could close this. The JIRA has 12.0.1.Final as fixed version.

**Pull request merged**